src/update-manager.ts

```diff
@@ -106,7 +106,14 @@ export default class UpdateManager {
     }
     uri = termValue(uri)
 
-    if ((uri as string).slice(0, 8) === 'file:///') {
+    // new logic for requests fron non-http resources
+    //   if not http(s) request
```

Comment:
**suggestion**: Instead of having a high-level comment here, let the the code speak by itself by moving all those parts to functions with a respective name